R/R-BayesianNetwork/Portfile

```diff
@@ -4,19 +4,19 @@ PortSystem          1.0
 PortGroup           R 1.0
 
 # GitHub version lags behind.
-R.setup             cran paulgovan BayesianNetwork 0.3
-revision            1
+R.setup             cran paulgovan BayesianNetwork 0.3.2
+revision            0
 categories-append   math
+platforms           {darwin any}
 maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
-license             {Apache-1.1 Apache-2}
+license             Apache
```

Comment:
from upstream's GitHub it appears the license is `Apache-2`

Reply:
https://github.com/paulgovan/BayesianNetwork/blob/ba31a3b844f0229542efa5a5f948fa612b4e91d0/DESCRIPTION#L12

Reply:
well, how did you decide which was is correct if there is a discrepancy between that file and the `LICENSE` file? It's either `Apache`, which for MacPorts means version 1 or `Apache-2`. Please clarify with upstream

Reply:
I take DESCRIPTION as a statement for official purposes, since that file is parsed by CRAN, and license there is set accordingly.
But this is not something I really care about. I am totally okay setting it to Apache-2.